test/execflags/jhh/job_prefix.numlocales

```diff
@@ -0,0 +1 @@
+1
```

Comment:
Note that 1 locale is the default, so you don't need a numlocales file here

Reply:
Well... let me double check that. It's possible that the presence of a numlocales file will mean this gets run in the `-multilocale` mode, which is what we tend to run on the systems where we actually use a launcher, so it might be useful for that reason

Reply:
I just copied the files Greg created to test the "dry-run" option. Maybe they aren't all necessary.